src/lib/proxy/proxy-repository.ts

```diff
@@ -67,6 +73,7 @@ export class ProxyRepository
     }
 
     getToggle(name: string): FeatureInterface {
+        //@ts-ignore (we must update the node SDK to allow undefined)
```

Comment:
Yes! I faced the same issue, but it might be a breaking change. I'll propose a PR to discuss because other methods take into account that finding something might yield undefined

Reply:
we should do it. Its not really a breaking to allow more (I think). If you look at the actual implementation I would almost consider that a bug-fix, but let not take the full discussion here 🗡️